force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -166,6 +166,16 @@ public inherited sharing class RD2_RecurringDonation {
         return this;
     }
 
+    /**
+    * @description Returns the annual value of this Recurring Donation based on the amount and schedule
+    * @return Decimal The annual value of this Recurring Donation
+    */
+    public Decimal getAnnualValue(){
+        Decimal yearlyFrequency = RD_Constants.PERIOD_TO_YEARLY_FREQUENCY.get(rd.npe03__Installment_Period__c);
+        Decimal installmentFrequency = rd.InstallmentFrequency__c;
+        return rd.npe03__Amount__c * (yearlyFrequency / installmentFrequency);
```

Comment:
This value gets weird in some edge cases (ex. donation every 3 weeks, or 17.33 donations per year), so we could take the Floor of the interval to get a consistent number, BUT the value provided here will be technically correct over the long term, since some years will have an extra donation.

Reply:
This does not count pausd months or future schedule changes.  What about using the CurrentYearValue or NextYearValue custom fields?

Reply:
There's some complexity with the timing of when those fields get updated, it's during a "second" update phase which does not trigger the TDTM class again. I'll discuss with Mary, but I believe the intention with this new field is less about "actual value" and more a quick calculation of what this donation will bring in during a normal year.